src/config/digitallyhappy/assets.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+return [
+    'cache_cdns' => true,
+    'cache_directory' => public_path('bassets'),
```

Comment:
Update on this, right now it might place assets in TWO bad places 😅 In my current project, where I've internalized assets both by browsing and by running `php artisan bassets:internalize` look where the assets have went:

<img width="499" alt="CleanShot 2023-02-02 at 20 25 03@2x" src="https://user-images.githubusercontent.com/1032474/216337071-bceac164-a5e5-403c-98b7-cff65835c5e5.png">

Bleah, so messy. Unless I'm missing something, I see no reason why they shouldn't be stored in `storage/bassets` - that would be so much cleaner. Let me know if you disagree @promatik

Reply:
Actually fixed in https://github.com/DigitallyHappy/assets/pull/19